### PR TITLE
Tweak Excite Truck Textures for consistency

### PR DESCRIPTION
**Pull request by teconmoon**
The Excite Truck textures being generated using the text-outlines style looked a bit odd, and some of the controller textures seen in the training mode were overlapping. This PR is intended to address those issues.

- Tweaked the GameID allowing compatibility with other regions
- Moved all texture coordinates to latest Defaultstyle JSON template
- Tweaked coordinates to improve display of generated "tilt" textures and avoid weird masking issues
- Added device textures for training mode
- Cleaned up unnecessary texture files

### CODE REVIEW

File: Excite Truck/UniversalDIT_Defaultstyle.json [hunk 1]
```diff
@@ -1432,122 +1407,6 @@
...
-      "`Axis 0-`": "../#DefaultDevices/DualShock4/Defaultstyle/Left X-.png",
-      "`Axis 0+`": "../#DefaultDevices/DualShock4/Defaultstyle/Left X+.png",
-      "`Axis 1-`": "../#DefaultDevices/DualShock4/Defaultstyle/Left Y+.png",
-      "`Axis 1+`": "../#DefaultDevices/DualShock4/Defaultstyle/Left Y-.png",
-      "`Axis 2-`": "../#DefaultDevices/DualShock4/Defaultstyle/Right X-.png",
-      "`Axis 2+`": "../#DefaultDevices/DualShock4/Defaultstyle/Right X+.png",
-      "`Axis 3+`": "../#DefaultDevices/DualShock4/Defaultstyle/Right Y-.png",
-      "`Axis 3-`": "../#DefaultDevices/DualShock4/Defaultstyle/Right Y+.png",
-      "`Full Axis 4+`": "../#DefaultDevices/DualShock4/Defaultstyle/L2.png",
-      "`Full Axis 5+`": "../#DefaultDevices/DualShock4/Defaultstyle/R2.png"
```
Comment (Venomalia) [inline Excite Truck/UniversalDIT_Defaultstyle.json:1550]: The changes to the devices must be reversed, these are not up to date.

**Review by Venomalia — CHANGES_REQUESTED** [review]
The changes to the devices must be reversed.

File: Excite Truck/UniversalDIT_Defaultstyle.json (hunk 1 above)
Comment (teconmoon) [reply Excite Truck/UniversalDIT_Defaultstyle.json:1550]: Hi there, I'm not sure how this drifted out-of-date, I just took the latest UniversalDIT_Defaultstyle.json file from the #Assets folder and used it as a base. Does this appear the same there?

Comment (teconmoon) [reply Excite Truck/UniversalDIT_Defaultstyle.json:1550]: @Venomalia I will approve the changes to push this through, but all of these devices that you've marked for deletion appear to exist under the UniversalDIT_Defaultstyle.json.

Comment (Venomalia) [reply Excite Truck/UniversalDIT_Defaultstyle.json:1550]: I have not marked them for deletion, this means that your changes will delete them when they are merged into the main package.

**Comment by teconmoon** [comment]
@Venomalia Sorry can you check it again? I tried re-copying the devices section from UniversalDIT_Defaultstyle.json in #Assets and pasting it over the Excite Truck JSON. I'm not really sure how it got changed. Sorry for the trouble.

**Review by Venomalia — APPROVED** [review]

**Comment by Venomalia** [comment]
there is the possibility to `squash` several commits into one which makes it clear to view for next time.

I simply did that now.

**Pull request merged**